CRM/Chfunds/Upgrader.php

```diff
@@ -53,6 +53,27 @@ public function upgrade_1400() {
     return TRUE;
   }
 
+  public function upgrade_1500() {
+    $this->ctx->log->info('Applying update 1.5');
+
+    $revenueFinancialAccountTypeID = array_search('Revenue', CRM_Core_OptionGroup::values('financial_account_type', FALSE, FALSE, FALSE, NULL, 'name'));
+
+    if ($revenueFinancialAccountTypeID) {
+      $sql = "
+        UPDATE civicrm_financial_account fa INNER JOIN (
+        SELECT fa.id, ft.name
+        FROM civicrm_financial_account fa
+        INNER JOIN civicrm_entity_financial_account efa ON efa.financial_account_id = fa.id AND fa.financial_account_type_id = $revenueFinancialAccountTypeID
+        INNER JOIN civicrm_financial_type ft ON ft.id = efa.entity_id AND efa.entity_table = 'civicrm_financial_type'
+        WHERE fa.name <> ft.name ) temp ON temp.id = fa.id
```

Comment:
NO, we don't want to make more changes than to the single case where the Financial Type was renamed from Donation to General Fund but the associated Financial Account was missed. So please change this line to:

WHERE fa.name='Donation' AND ft.name='General Fund' ) temp ON temp.id = fa.id